RE-Cryptography/Sessions-5-&-6/krishna/krishna-ceaser_encryption.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+
+void main()
+{
+    char str[500];
+    FILE* f1 = fopen("/home/akash/akash-message-caesar.txt","r");
```

Comment:
Indicates that you did not do this assignment in true honesty. Sad to see. Write original code.

Reply:
You are supposed to do Phase 1 individually